src/codegen/ast_interpreter.h

```diff
@@ -33,6 +34,41 @@ struct LineInfo;
 
 extern const void* interpreter_instr_addr;
 
+struct ASTInterpreterJitInterface {
+    static int getCurrentBlockOffset();
+    static int getCurrentInstOffset();
+
+    static Box* derefHelper(void* interp, InternedString s);
+    static Box* doOSRHelper(void* interp, AST_Jump* node);
+    static Box* getBoxedLocalHelper(void* interp, BoxedString* s);
+    static Box* getBoxedLocalsHelper(void* interp);
+    static Box* getLocalHelper(void* interp, InternedString id);
+    static Box* landingpadHelper(void* interp);
+    static Box* setExcInfoHelper(void* interp, Box* type, Box* value, Box* traceback);
+    static Box* uncacheExcInfoHelper(void* interp);
+    static Box* yieldHelper(void* interp, Box* val);
+    static void setItemNameHelper(void* interp, Box* str, Box* val);
+    static void setLocalClosureHelper(void* interp, InternedString id, Box* v);
+    static void setLocalHelper(void* interp, InternedString id, Box* v);
+};
```

Comment:
Not sure if I'm a huge fan of this :P  I can see the point though and maybe we can find a better solution later -- I think we'll probably see some pretty tight coupling between the baseline jit and the interpreter, so maybe it makes sense to make these members of AstInterpreter and then make that class visible to the baseline jit?